src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -61,6 +61,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
+        Private ReadOnly Property IIsInvalid As Boolean Implements IOperation.IsInvalid
```

Comment:
This case follows a naming convention for implementations of IOperation members, prefacing the implemented member name with "I". One might question the appropriateness of the convention, but addressing that would be a separate change set.